exp/window_func_test.go

```diff
@@ -151,31 +151,31 @@ func (swfet *sqlWindowFunctionExpressionTest) TestAllOthers() {
 
 	expIsNull := wf.IsNull()
 	swfet.Equal(expIsNull.LHS(), wf)
-	swfet.Equal(expIsNull.Op(), IsOp)
+	swfet.Equal(expIsNull.Op(), EqOp)
```

Comment:
The is methods should work like they did previously. Take a look at https://stackoverflow.com/questions/9822154/standard-sql-boolean-operator-is-vs-equals-operator it explains the difference pretty well.